.github/workflows/go-checks.yml

```diff
@@ -28,37 +26,30 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
-      - name: Install Go
-        uses: actions/setup-go@v3
+      - uses: actions/setup-go@v3
         with:
-          go-version: ${{ env.GO_VERSION }}
+          go-version-file: src/go.mod
```

Comment:
Big plus for this, as one using this for personal `golang` projects.

Reply:
💯